data/categories/ae_arts_entertainment.yml

```diff
@@ -7740,6 +7743,201 @@
   - instrument_bow_material
   - instrument_material
   - string_instrument_size
+- id: ae-2-8-7-7
+  name: Banjos
+  children:
+  - ae-2-8-7-7-1
+  - ae-2-8-7-7-2
+  - ae-2-8-7-7-3
+  - ae-2-8-7-7-4
+  attributes:
+  - instrument_color
+  - instrument_material
+  - instrument_pattern
+  - instrument_string_color
+  - instrument_string_material
+  - instrument_string_pattern
+- id: ae-2-8-7-7-1
+  name: 4-String Banjos
+  children: []
+  attributes:
+  - instrument_color
+  - instrument_material
+  - instrument_pattern
+  - instrument_string_color
+  - instrument_string_material
+  - instrument_string_pattern
+- id: ae-2-8-7-7-2
+  name: 5-String Banjoss
+  children: []
+  attributes:
+  - instrument_color
+  - instrument_material
+  - instrument_pattern
+  - instrument_string_color
+  - instrument_string_material
+  - instrument_string_pattern
+- id: ae-2-8-7-7-3
+  name: 6-String Banjos
+  children: []
+  attributes:
+  - instrument_color
+  - instrument_material
+  - instrument_pattern
+  - instrument_string_color
+  - instrument_string_material
+  - instrument_string_pattern
+- id: ae-2-8-7-7-4
+  name: Tenor Banjos
+  children: []
+  attributes:
+  - instrument_color
+  - instrument_material
+  - instrument_pattern
+  - instrument_string_color
+  - instrument_string_material
+  - instrument_string_pattern
```

Comment:
Tenor banjos typically have four strings, so I would skip `Tenor Banjos` and keep `4-String Banjos`, `5-String Banjos` and `6-String Banjos` to avoid overlaps.
![image](https://github.com/user-attachments/assets/0378eb07-8dda-4671-8ef7-79284ffda133)